CorePayments/src/main/java/com/paypal/android/corepayments/RestClient.kt

```
@@ -40,8 +40,10 @@ class RestClient internal constructor(
             "Accept-Language" to language
         )
 
-        accessToken?.let { token ->
-            headers["Authorization"] = "Bearer $token"
+        if (clientId != null) {
```

Do we want to throw an error here if the client ID is null?

---

This is a good call. It's non-null in `CoreConfig` so this won't happen. I'll refactor to update.